configure.ac

```diff
@@ -341,6 +341,40 @@ if test "x$enable_btrfs" = "xyes" \
 fi
 AM_CONDITIONAL(HAVE_BTRFS, [test "x$have_btrfs" = "xyes"])
 
+# loop module
+have_loop=no
+AC_ARG_ENABLE(loop, AS_HELP_STRING([--enable-loop], [enable loop support]))
```

Comment:
This is still wrong. There is check for blockdev swap, mdraid and fs plugins needed by udisks core on line 474, you should use the same approach for loop.